lte/gateway/python/magma/pipelined/app/egress.py

```diff
@@ -315,8 +309,8 @@ def _monitor_and_update(self):
             gw_info_list = get_mobilityd_gw_info()
             for gw_info in gw_info_list:
                 if gw_info and gw_info.ip:
-                    latest_mac_addr = self._get_gw_mac_address(gw_info.ip, gw_info.vlan)
-                    if latest_mac_addr is None or latest_mac_addr == "":
+                    latest_mac_addr = get_gw_mac_address(gw_info.ip, gw_info.vlan, self.config.non_nat_arp_egress_port)
+                    if latest_mac_addr == "":
```

Comment:
nit: if None condition is removed the `get_gw_mac_address` should have an if-else structure (yes, we do expect that either ipv4 or ipv6 is the case)

Reply:
`get_gw_mac_address` does return empty string now in neither ipv4 nor ipv6 case to be consistent with the return type hint.